README.md

```diff
@@ -355,6 +355,19 @@ Mode for config_file.
 
 - *Default*: '0640'
 
+config_file_contents
+--------------------
+Limits that should be placed into limits.conf
```

Comment:
`Ordered array of limits that should be placed into limits.conf. Useful for Suse 10 which does not use limits.d.`